### ISSUE TITLE
No offline mode when server unavailable

### ISSUE THREAD

**Issue by ivanboytsov** [emansih/FireflyMobile#117]
**Firefly III Server Version:**
5.3.3
**Firefly III Mobile Version:**
3.0.1
**Android Version:**  
9
**Device Information:**  

- BRAND:samsung
- MODEL: j7
--------------------------------------

## Expected Behavior
App let's me use adding transactions offline both when no internet connection and when server unavailable.
## Current Behavior
No offline mode available only when no connection to the Internet. If server is down the app just stuck in loading.

## Steps to Reproduce
1. Turn off server.
2. Add transaction.
3. Loading of adding form stuck.

**Comment by emansih** [comment]
I can't replicate this behavior. The loading circle should disappear even if there is no internet connection

**Comment by ivanboytsov** [comment]
Tested again. Now the form loads, but I am getting error after adding the transaction.
[Fireflyiii-mobile.txt](https://github.com/emansih/FireflyMobile/files/5178358/Fireflyiii-mobile.txt)

**Comment by emansih** [comment]
yeah, I have fixed the bug. Will be available in the next release
